src/l10n.json

```diff
@@ -238,6 +238,7 @@
     "comments.status.delbyclass": "Deleted by class",
     "comments.status.hiddenduetourl": "Hidden due to URL",
     "comments.status.markedbyfilter": "Marked by filter",
+    "comments.status.censbyunconstructive": "Censored unconstructive",
```

Comment:
could this (and others above, ideally) name use camelCase instead of all-lowercase? camelCase seems to be the norm above...

Reply:
Yeah they are that way because they being concatted directly from the server response